package.json

```diff
@@ -41,7 +41,7 @@
     "@ladjs/graceful": "^1.0.5",
     "@sentry/node": "^6.17.4",
     "@sentry/tracing": "^6.17.4",
-    "@yfi/sdk": "^1.0.0",
+    "@yfi/sdk": "^1.2.1-0",
```

Comment:
Need to update this to v2.0 when it is released, just using this pre-release for now

Reply:
going with 1.2.1-1